test/graph_test.py

```diff
@@ -205,7 +205,8 @@ def test_split():
                         transaction_id)
 
     second_graph = ursa.graph.Graph.remote(transaction_id,
-                                           graph.split.remote())
+                                           key,
```

Comment:
What is this `key`?

Reply:
This key is being passed in for the graph_id. This is just used as a placeholder since this test does not actually use the localManager. I've commented below reg'd why I pass in the graph_id here.